packages/wonder-blocks-modal/building-blocks.md

```diff
@@ -1 +1,134 @@
 Use these low-level building blocks to build your own modal dialog, instead of using OnePaneDialog. This should happen very rarely and only when a specific exception is required.
+
+### Example: Custom Two-Pane Dialog
+
+```js
+const {StyleSheet} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {Title, Body} = require("@khanacademy/wonder-blocks-typography");
+const {MediaLayout, Strut} = require("@khanacademy/wonder-blocks-layout");
+
+const styles = StyleSheet.create({
+    previewSizer: {
+        height: 512,
+    },
+
+    modalPositioner: {
+        // Checkerboard background
+        backgroundImage: "linear-gradient(45deg, #ccc 25%, transparent 25%), linear-gradient(-45deg, #ccc 25%, transparent 25%), linear-gradient(45deg, transparent 75%, #ccc 75%), linear-gradient(-45deg, transparent 75%, #ccc 75%)",
+        backgroundSize: "20px 20px",
+        backgroundPosition: "0 0, 0 10px, 10px -10px, -10px 0px",
+
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "center",
+
+        position: "absolute",
+        left: 0,
+        right: 0,
+        top: 0,
+        bottom: 0,
+    },
+});
+
+const styleSheets = {
+    mdOrLarger: StyleSheet.create({
+        dialog: {
+            width: "86.72%",
+            maxWidth: 888,
+            height: "60.42%",
+            minHeight: 308,
+        },
+
+        row: {
+            flexDirection: "row",
+            flex: 1,
+        },
+
+        below: {
+            background: "url(/blue-blob.png)",
+            backgroundSize: "cover",
+            width: 294,
+            height: 306,
+            position: "absolute",
+            top: 100,
+            left: -60
+        },
+
+        above: {
+            background: "url(/asteroid.png)",
+            backgroundSize: "cover",
+            width: 650,
+            height: 400,
+            position: "absolute",
+            top: -35,
+            left: 50
+        },
+    }),
+
+    small: StyleSheet.create({
+        dialog: {
+            width: "100%",
+            height: "100%",
+            overflow: "hidden",
+        },
+
+        row: {
+            flexDirection: "column",
```

Comment:
Nit: this is a little weird.  It might make sense to call this style `column` and then use `mediaSize` to decide whether to use `style.row` or `style.column`.

Reply:
To avoid adding extra logic in the examples, I'm going to use a more generic name: `panelGroup`.